src/ngraph/runtime/backend.cpp

```diff
@@ -62,3 +62,28 @@ bool runtime::Backend::is_supported_property(const Property prop) const
 void runtime::Backend::remove_compiled_function(std::shared_ptr<Executable> exec)
 {
 }
+
+std::shared_ptr<ngraph::runtime::Allocator> runtime::Backend::get_framework_memory_allocator()
+{
+    return nullptr;
+}
+
+void runtime::Backend::set_framework_memory_allocator(
+    const std::shared_ptr<ngraph::runtime::Allocator>& allocator)
+{
+}
+
+ngraph::runtime::AllocateFunc runtime::Backend::get_device_memory_alloc()
+{
+    return nullptr;
+}
+
+ngraph::runtime::DestroyFunc runtime::Backend::get_device_memory_dealloc()
+{
+    return nullptr;
+}
+
+bool runtime::Backend::is_device_memory()
+{
+    return false;
+}
```

Comment:
It looks like we want to make all these methods pure virtual so that they are implemented by the subclasses, right?